cumulus/parachains/runtimes/testing/penpal/src/lib.rs

```diff
@@ -130,11 +130,34 @@ pub type SignedExtra = (
 pub type UncheckedExtrinsic =
 	generic::UncheckedExtrinsic<Address, RuntimeCall, Signature, SignedExtra>;
 
+#[rustfmt::skip]
 pub type Migrations = (
-	pallet_balances::migration::MigrateToTrackInactive<Runtime, xcm_config::CheckingAccount>,
-	pallet_collator_selection::migration::v1::MigrateToV1<Runtime>,
+	migrations::V1_05_00,
```

Comment:
Penpal is a test network we actually deploy and maintain?

Reply:
It is not used much afaik but it is easier to have the versioning sorted for **all** runtimes.